src/core/accido/_class_regularword.py

```
@@ -5,11 +5,14 @@
 import logging
 from functools import total_ordering
 from typing import TYPE_CHECKING
+from warnings import deprecated
```

Pylint (3.3.7)

[error] No name 'deprecated' in module 'warnings'

(E0611)